index.js

```diff
@@ -1,80 +1,52 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import select from './lib/select/index';
-
-window.React = React;
-
-window.Styleguide = {
-  "select": select
-}
-
-window.Styleguide.create = (opts, elem) => {
-  if(!window.Styleguide[opts.component]) {
-    return;
+import ComponentPlayground from 'component-playground';
+import Playground from './lib/react/playground/index';
+
+import Select from './lib/react/select/index';
+import EmptyState from './lib/react/empty-state/index';
+import LoadingPanel from './lib/react/loading-panel/index';
+
+const SelectExample = require('raw!./lib/react/select/example.txt');
+const LoadingPanelExample = require('raw!./lib/react/loading-panel/example.txt');
+const EmptyStateExample = require('raw!./lib/react/empty-state/example.txt');
+
+class ReactComponents extends React.Component {
+  render() {
+    return (
+      <section className="section content-styleguide">
+
+        <Playground title="Select" description="Select component description text.">
```

Comment:
Playground is repeated here. Maybe you can use an array with the data and map each item instead? 

`let playgroundList = [{
title: 'select',
description: 'select component description text.',
codeText: SelectExample, 
(...)
}, {...}]
`

and then 

`
playgroundList.map(item => {
return (
<Playground title={item.title} description={item.description}>
  <ComponentPlayground  (...) />
<Playground/>);
})
`